src/java/org/apache/cassandra/config/Schema.java

```diff
@@ -333,6 +333,14 @@ public List<String> getUserKeyspaces()
         return ImmutableList.copyOf(Sets.difference(getNonSystemKeyspacesSet(), replicatedSystemKeyspaceNames));
     }
 
+    /**
+     * @return collection of the user defined keyspaces excluding system_palantir
+     */
+    public List<String> getUserKeyspacesPalantir()
```

Comment:
is there a name we can use that doesn't reference palantir? like getNonAdminKeyspaces or something? then we can also replace the equivalent in StorageService::areKeyspacesEmptyForBootstrap